### PR TITLE
Add course notes to course detail view

### LINKED ISSUE #2960: Show course notes in course detail
Stemming from #2952, course list notes were added to the list rows. We should show them in the detail view as well.

### PR DESCRIPTION
**Pull request by drewvolz**
Closes #2960 

Keeps the same style as the Description field. Lists the notes above the Description field.

### CODE REVIEW

**Review by rye — DISMISSED** [review]

**Review by hannesmcman — DISMISSED** [review]

File: source/views/sis/course-search/detail/index.js [hunk 1]
```diff
@@ -139,6 +139,24 @@ function Schedule({course}: {course: CourseType}) {
 	)
 }
 
+function Notes({course}: {course: CourseType}) {
+	let notesText = course.notes ? course.notes.join(' ') : ''
```
Comment (hawkrives) [inline source/views/sis/course-search/detail/index.js]: This should be joined on newline (same on the list view) IMO

File: source/views/sis/course-search/detail/index.js [hunk 2]
```diff
@@ -139,6 +139,24 @@ function Schedule({course}: {course: CourseType}) {
...
+			<Cell
+				cellContentView={
+					<Text selectable={true} style={styles.chunk}>
+						{notesText}
+					</Text>
+				}
+			/>
+		)
+
+	return course.notes ? <Section header="NOTES">{notes}</Section> : null
```
Comment (hawkrives) [inline source/views/sis/course-search/detail/index.js]: At the top: "if not notesText: return null" instead of creating the empty objects.

**Review by hawkrives — CHANGES_REQUESTED** [review]

File: source/views/sis/course-search/detail/index.js (hunk 1 above)
Comment (drewvolz) [reply source/views/sis/course-search/detail/index.js]: I'll leave it up to you, but I think spaces looks better.

<details>
<summary>Screenshots</summary>

Spaces | Newlines
--|--
android-space2 | android-newline2
android-space | android-newline

</details>

**Review by hawkrives — APPROVED** [review]

**Comment by hawkrives** [comment]
Apparently I just merged that

**Comment by hawkrives** [comment]
Apparently approve + merge is very much a force of habit

**Pull request merged**